bin/node/testing/src/bench.rs

```diff
@@ -297,9 +298,8 @@ impl BenchDb {
 			&self.keyring,
 		);
 
-		let version = client.runtime_version_at(&BlockId::number(0))
-			.expect("There should be runtime version at 0")
-			.spec_version;
+		let RuntimeVersion { spec_version, transaction_version, .. } = client.runtime_version_at(&BlockId::number(0))
```

Comment:
You can also prevent the import of `sp-version` by calling `version.spec_version` & `version.transaction_version`.